scripts/zones/Grand_Palace_of_HuXzoi/mobs/Ixaern_MNK.lua

```diff
@@ -7,6 +7,38 @@ local ID = zones[xi.zone.GRAND_PALACE_OF_HUXZOI]
 ---@type TMobEntity
 local entity = {}
 
+local bracerMode = function(mob, qnAern1, qnAern2)
+    -- Hundred Fists
+    mob:useMobAbility(xi.jsa.HUNDRED_FISTS)
+    mob:addMod(xi.mod.ATT, 200)
+    -- captures show delay reduction from 280 -> 120
+    -- note this is actual delay reduction with change in tp gained and imparted
+    -- note lvl 83 mnk with martial arts vii
+    mob:setMod(xi.mod.DELAY, 2600)
+
+    if qnAern1 and qnAern1:isAlive() then
+        qnAern1:setAnimationSub(2)
+        qnAern1:addMod(xi.mod.ATT, 200)
+        -- captures show delay reduction from 240 -> 120
+        -- note this is actual delay reduction with change in tp gained and imparted
+        qnAern1:setMod(xi.mod.DELAY, 2000)
+    end
+
+    if qnAern2 and qnAern2:isAlive() then
+        qnAern2:setAnimationSub(2)
+        qnAern2:addMod(xi.mod.ATT, 200)
+        -- captures show delay reduction from 240 -> 120
+        -- note this is actual delay reduction with change in tp gained and imparted
+        qnAern2:setMod(xi.mod.DELAY, 2000)
+    end
+
+    -- slightly delay adding local var to avoid adding bracers to Ix'Mnk
+    -- while Hundred Fists is still active.
+    mob:timer(3000, function(mobArg)
+        mobArg:setLocalVar('enableBracerAnimation', 1)
+    end)
+end
+
 entity.onMobInitialize = function(mob)
     mob:addListener('ITEM_DROPS', 'ITEM_DROPS_IXAERN_MNK', function(mobArg, loot)
         local rate = mob:getLocalVar('[SEA]IxAern_DropRate')
```

Comment:
> Unsure if you want to make the change on the drop rate here or if I am overstepping in suggesting it. From what I can gleam in the available data between FFXIDB and the user reports on Clopedia. I want to say the capes from the various Ix'Aerns are a 15% drop rate and not 25%.
> 
> This would then also correspond to the official drop rates of Common, Uncommon, and Rare when trading 3, 2 or 1 organs.

Yeah you are probably correct but I think better if someone puts such rate changes in a separate PR so they are all changed and harmonized at once.